v1.1.yml

```diff
@@ -1591,3 +1767,259 @@ definitions:
       email:
         type: string
         x-nullable: true
+
+  StudentObject:
```

Comment:
It's because of the return format of our events (which is quite verbose...)
```
data : {
  data : {
    object : {}
    additional_attributes: {}
  }
}
```
We need the wrap in an object to take care of the `object` field in the response. It's worth thinking about whether we can clean this up giving that it makes the Java code a bit verbose, but it does represent our response schema.